src/Plugin/client_error/ReportInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\client_error_trace\Plugin\client_error;
+
+/**
+ * Interface for all ClientError reports.
+ *
+ * @package Drupal\client_error_trace\Plugin\client_error
+ */
+interface ReportInterface {
+
+  /**
+   * The ClientError test failed.
+   *
+   * @var int
+   */
+  const FAILED = 0;
```

Comment:
I'm not a fan of sequential numeric constants in extendable code. But I can't think of a case where `CoolReportInterface` would add a constant with value :three: So this seems to be a closed set of constants.

:+1: